include/fcl/BV/RSS.h

```diff
@@ -111,7 +111,7 @@ class RSS
   /// @brief Volume of the RSS
   inline FCL_REAL volume() const
   {
-    return (l[0] * l[1] * 2 * r + 4 * boost::math::constants::pi<FCL_REAL>() * r * r * r);
+    return (l[0] * l[1] * 2 * r + 4 * M_PI * r * r * r);
```

Comment:
Two problems with `M_PI`:
- it is a macro and hence untyped; needs to have type FCL_REAL
- it is non-standard (it is a Posix feature, not a C++11 one)

Since "pi" is the only boost math constant FCL used, I think the simplest thing would be to define an fcl::kPi or whatever that could be defined in a common header with something like
```
constexpr FCL_REAL kPi=FCL_REAL(/*lots of digits of pi copied from a reliable source*/);
```